tests/e2e/test_parameterized_queries.py

```diff
@@ -173,185 +278,10 @@ def test_use_inline_by_default_with_warning(self, explicit_inline, caplog):
                             "Consider using native parameters." in caplog.text
                         ), "Log message should not be supressed"
 
-    @both_approaches
```

Comment:
Why are we removing all of these?

Reply:
We're not...

Reply:
It's just replaced with an extra dimension to the matrix, as described in the PR description.

https://github.com/databricks/databricks-sql-python/blob/a86abd35fe525676369d49b9124bb1c2aaa19fbb/tests/e2e/test_parameterized_queries.py#L52-L56